File: Entities/DepartmentEn.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.ServiceModel;
using System.Runtime.Serialization;

namespace HTS.SAS.Entities
{
    [System.SerializableAttribute()]
    //DataContract
    public class DepartmentEn
    {
        private int ciAutoID;
        private string csDepartmentID;
        private string csDepartment;
        private bool cbStatus;
        private string csCreatedBy;
        private DateTime coCreateDate;
        private string csModifiedBy;
        private DateTime coModifiedDate;
        private string csSqlCase;
        private string csSearchCriteria;

        [System.Xml.Serialization.XmlElement]
        ////[DataMember]
        public int AutoID
        {
            get { return ciAutoID; }
            set { ciAutoID = value; }
        }

        [System.Xml.Serialization.XmlElement]
        ////[DataMember]
        public string DepartmentID
        {
            get { return csDepartmentID; }
            set { csDepartmentID = value; }
        }

        [System.Xml.Serialization.XmlElement]
        ////[DataMember]
        public string Department
        {
            get { return csDepartment; }
            set { csDepartment = value; }
        }

        [System.Xml.Serialization.XmlElement]
        ////[DataMember]
        public bool Status
        {
            get { return cbStatus; }
            set { cbStatus = value; }
        }

        [System.Xml.Serialization.XmlElement]
        ////[DataMember]
        public string CreatedBy
        {
            get { return csCreatedBy; }
            set { csCreatedBy = value; }
        }

        [System.Xml.Serialization.XmlElement]
        ////[DataMember]
        public DateTime CreateDate
        {
            get { return coCreateDate; }
            set { coCreateDate = value; }
        }

        [System.Xml.Serialization.XmlElement]
        ////[DataMember]
        public string ModifiedBy
        {
            get { return csModifiedBy; }
            set { csModifiedBy = value; }
        }

        [System.Xml.Serialization.XmlElement]
        ////[DataMember]
        public DateTime ModifiedDate
        {
            get { return coModifiedDate; }
            set { coModifiedDate = value; }
        }

        [System.Xml.Serialization.XmlElement]
        ////[DataMember]
        public string SqlCase
        {
            get { return csSqlCase; }
            set { csSqlCase = value; }
        }

        [System.Xml.Serialization.XmlElement]
        ////[DataMember]
        public string SearchCriteria
        {
            get { return csSearchCriteria; }
            set { csSearchCriteria = value; }
        }

    }

    
}

File: BusinessEntities/DepartmentEn.vb
Public Class DepartmentEn

    Private _AutoID As Integer
    Public Property AutoID() As Integer
        Get
            Return _AutoID
        End Get
        Set(ByVal value As Integer)
            _AutoID = value
        End Set
    End Property

    Private _DepartmentID As String
    Public Property DepartmentID() As String
        Get
            Return _DepartmentID
        End Get
        Set(ByVal value As String)
            _DepartmentID = value
        End Set
    End Property

    Private _Department As String
    Public Property Department() As String
        Get
            Return _Department
        End Get
        Set(ByVal value As String)
            _Department = value
        End Set
    End Property

    Private _Status As Boolean
    Public Property Status() As Boolean
        Get
            Return _Status
        End Get
        Set(ByVal value As Boolean)
            _Status = value
        End Set
    End Property

    Private _CreatedBy As String
    Public Property CreatedBy() As String
        Get
            Return _CreatedBy
        End Get
        Set(ByVal value As String)
            _CreatedBy = value
        End Set
    End Property

    Private _CreateDate As DateTime
    Public Property CreateDate() As DateTime
        Get
            Return _CreateDate
        End Get
        Set(ByVal value As DateTime)
            _CreateDate = value
        End Set
    End Property

    Private _ModifiedBy As String
    Public Property ModifiedBy() As String
        Get
            Return _ModifiedBy
        End Get
        Set(ByVal value As String)
            _ModifiedBy = value
        End Set
    End Property

    Private _ModifiedDate As DateTime
    Public Property ModifiedDate() As DateTime
        Get
            Return _ModifiedDate
        End Get
        Set(ByVal value As DateTime)
            _ModifiedDate = value
        End Set
    End Property

    Private _SQLCase As String
    Public Property SQLCase() As String
        Get
            Return _SQLCase
        End Get
        Set(ByVal value As String)
            _SQLCase = value
        End Set
    End Property

    Private _SearchCriteria As String
    Public Property SearchCriteria() As String
        Get
            Return _SearchCriteria
        End Get
        Set(ByVal value As String)
            _SearchCriteria = value
        End Set
    End Property

End Class
